docs/design/latency_definitions/node.md

```diff
@@ -0,0 +1,58 @@
+# Node
+
+Node latency is the time difference between when a callback takes ownership and when it abandons.
+
+$$
+l_{node} = t_{transfer} - t_{receive}
+$$
+
+In particular, if transfer of ownership is subscribe and receive of ownership is publish, it is defined as follows
+
+$$
+l_{node} = t_{pub} - t_{sub}
+$$
+
+See [Communication](./communication.md) for the reason ownership is used to define latency.
+
+## Message context
+
+A node receives a message, processes it, and then publishes it to a subsequent node.  
+The dependency between the subscribed and published messages is used to define node latency.  
+In the following sections explain the message dependency (message context), which is the concept of node latency.
+
+For example, consider the following callback.
+
+```c++
+auto subsription_callback = [](&msg){
+        msg_ = f(msg);
+        pub.publish(msg_);
+}
+```
+
+In this case, the received message is processed and published immediately.  
+The dependency of the message at this time is described in chronological order and is expressed as follows.
+
+![message context](../../imgs/message_context.drawio.png)
+
+Here, the time difference between Subscribe and Publish can be calculated as node latency.  
+In this way, node latency can be calculated when message dependencies are defined.
+6
+In the previous example, we presented a case where a node is subscribed and immediately published.
+In actuality, the Message context can be quite complex, because node processing can be defined by the developer.
```

Comment:
```suggestion
Actually, the message context can be quite complex because data processing in a node is freely defined by an application developer.
```